src/Form/WebdamConfig.php

```diff
@@ -29,10 +63,28 @@ protected function getEditableConfigNames() {
   }
 
   /**
+   * Checks if we can get subscription details.
+   *
+   * @return bool
+   *   Whether client is authenticated or not.
+   */
+  protected function isAuthenticated() {
+    if ($this->webdam->getSubscriptionDetails()) {
```

Comment:
Can we make this a bit more robust?

For instance, something like this:

```php
$subscription_details = $this->webdam->getSubscriptionDetails();
return isset($subscription_details->url);
```

Or something like that. That way, we're checking the contents of the object, rather than whether or not an object exists at all.